content/configuration/dev-server.md

```diff
@@ -208,6 +209,21 @@ To disable `contentBase`:
 contentBase: false
 ```
 
+### `devServer.watchContentBase`
+
+`boolean`
+
+Tell the server to watch the files served by the contentBase option. File changes will trigger a full page reload.
+
+It is possible to enable this option by setting it to true:
```

Comment:
Could you remove this sentence? The code sample already makes clear how to enable it and that way it's more consistent with the other options.